app/views/works/_filters.html.erb

```diff
@@ -162,6 +162,12 @@
       <dd class="submit actions"><%= f.submit ts("Sort and Filter") %></dd>
     </dl>
 
+    <% if @pseud || @user || @collection || @tag %>
```

Comment:
You can use `@owner` here:

https://github.com/otwcode/otwarchive/blob/6551990716b90f441ce9cbbb476a42a6d4142185/app/controllers/works_controller.rb#L773